From 225f994bfda8fd42ec875acb4a49e27445aefe03 Mon Sep 17 00:00:00 2001
From: Jenkins <dev-kitchen@cloudera.com>
Date: Wed, 10 Oct 2018 03:21:14 -0700
Subject: [PATCH 480/489] Branching for 5.16.1-SNAPSHOT on Wed Oct 10 03:20:51
 PDT 2018

JOB_NAME : 'Cut-Release-Branches'
BUILD_NUMBER : '546'
CODE_BRANCH : ''
OLD_CDH_BRANCH : 'cdh5'

Pushed to remote apache	git@github.sf.cloudera.com:Mirrors/apache-hbase.git (push)
origin	git@github.sf.cloudera.com:CDH/hbase.git (push)
---
 hbase-annotations/pom.xml                          |    2 +-
 hbase-assembly/pom.xml                             |    2 +-
 hbase-checkstyle/pom.xml                           |    4 ++--
 hbase-client/pom.xml                               |    2 +-
 hbase-common/pom.xml                               |    2 +-
 hbase-examples/pom.xml                             |    2 +-
 hbase-external-blockcache/pom.xml                  |    2 +-
 hbase-hadoop-compat/pom.xml                        |    2 +-
 hbase-hadoop2-compat/pom.xml                       |    2 +-
 hbase-it/pom.xml                                   |    2 +-
 hbase-prefix-tree/pom.xml                          |    2 +-
 hbase-procedure/pom.xml                            |    2 +-
 hbase-protocol/pom.xml                             |    2 +-
 hbase-resource-bundle/pom.xml                      |    2 +-
 hbase-rest/pom.xml                                 |    2 +-
 hbase-rsgroup/pom.xml                              |    2 +-
 hbase-server/pom.xml                               |    2 +-
 hbase-shaded/hbase-shaded-check-invariants/pom.xml |    2 +-
 hbase-shaded/hbase-shaded-client/pom.xml           |    2 +-
 hbase-shaded/hbase-shaded-server/pom.xml           |    2 +-
 hbase-shaded/pom.xml                               |    2 +-
 hbase-shell/pom.xml                                |    2 +-
 hbase-spark/pom.xml                                |    2 +-
 hbase-testing-util/pom.xml                         |    2 +-
 hbase-thrift/pom.xml                               |    2 +-
 pom.xml                                            |    4 ++--
 26 files changed, 28 insertions(+), 28 deletions(-)

diff --git a/hbase-annotations/pom.xml b/hbase-annotations/pom.xml
index 964941e..4be353e 100644
--- a/hbase-annotations/pom.xml
+++ b/hbase-annotations/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-assembly/pom.xml b/hbase-assembly/pom.xml
index 4362c4f..1edbd4d 100644
--- a/hbase-assembly/pom.xml
+++ b/hbase-assembly/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
   <artifactId>hbase-assembly</artifactId>
diff --git a/hbase-checkstyle/pom.xml b/hbase-checkstyle/pom.xml
index 3a02e8e..8e59aa0 100644
--- a/hbase-checkstyle/pom.xml
+++ b/hbase-checkstyle/pom.xml
@@ -24,14 +24,14 @@
 <modelVersion>4.0.0</modelVersion>
 <groupId>org.apache.hbase</groupId>
 <artifactId>hbase-checkstyle</artifactId>
-<version>1.2.0-cdh5.16.0-SNAPSHOT</version>
+<version>1.2.0-cdh5.16.1-SNAPSHOT</version>
 <name>Apache HBase - Checkstyle</name>
 <description>Module to hold Checkstyle properties for HBase.</description>
 
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-client/pom.xml b/hbase-client/pom.xml
index b088364..bc0b0b3 100644
--- a/hbase-client/pom.xml
+++ b/hbase-client/pom.xml
@@ -24,7 +24,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-common/pom.xml b/hbase-common/pom.xml
index 2f1e630..c303370 100644
--- a/hbase-common/pom.xml
+++ b/hbase-common/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-examples/pom.xml b/hbase-examples/pom.xml
index a519ca3..bbff15e 100644
--- a/hbase-examples/pom.xml
+++ b/hbase-examples/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
   <artifactId>hbase-examples</artifactId>
diff --git a/hbase-external-blockcache/pom.xml b/hbase-external-blockcache/pom.xml
index 11b9d5f..4fcf433 100644
--- a/hbase-external-blockcache/pom.xml
+++ b/hbase-external-blockcache/pom.xml
@@ -25,7 +25,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
   <artifactId>hbase-external-blockcache</artifactId>
diff --git a/hbase-hadoop-compat/pom.xml b/hbase-hadoop-compat/pom.xml
index edea757..84b474c 100644
--- a/hbase-hadoop-compat/pom.xml
+++ b/hbase-hadoop-compat/pom.xml
@@ -23,7 +23,7 @@
     <parent>
         <artifactId>hbase</artifactId>
         <groupId>org.apache.hbase</groupId>
-        <version>1.2.0-cdh5.16.0-SNAPSHOT</version>
+        <version>1.2.0-cdh5.16.1-SNAPSHOT</version>
         <relativePath>..</relativePath>
     </parent>
 
diff --git a/hbase-hadoop2-compat/pom.xml b/hbase-hadoop2-compat/pom.xml
index b5fee5f..154857f 100644
--- a/hbase-hadoop2-compat/pom.xml
+++ b/hbase-hadoop2-compat/pom.xml
@@ -21,7 +21,7 @@ limitations under the License.
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-it/pom.xml b/hbase-it/pom.xml
index 2b3fc05..9f02569 100644
--- a/hbase-it/pom.xml
+++ b/hbase-it/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-prefix-tree/pom.xml b/hbase-prefix-tree/pom.xml
index 03947db..768d1b6 100644
--- a/hbase-prefix-tree/pom.xml
+++ b/hbase-prefix-tree/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-procedure/pom.xml b/hbase-procedure/pom.xml
index 21b4da8..d1ba1f0 100644
--- a/hbase-procedure/pom.xml
+++ b/hbase-procedure/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-protocol/pom.xml b/hbase-protocol/pom.xml
index 08e4b03..3a23480 100644
--- a/hbase-protocol/pom.xml
+++ b/hbase-protocol/pom.xml
@@ -23,7 +23,7 @@
     <parent>
         <artifactId>hbase</artifactId>
         <groupId>org.apache.hbase</groupId>
-        <version>1.2.0-cdh5.16.0-SNAPSHOT</version>
+        <version>1.2.0-cdh5.16.1-SNAPSHOT</version>
         <relativePath>..</relativePath>
     </parent>
 
diff --git a/hbase-resource-bundle/pom.xml b/hbase-resource-bundle/pom.xml
index ed9650d..2942abd 100644
--- a/hbase-resource-bundle/pom.xml
+++ b/hbase-resource-bundle/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-rest/pom.xml b/hbase-rest/pom.xml
index feb99c3..8052604 100644
--- a/hbase-rest/pom.xml
+++ b/hbase-rest/pom.xml
@@ -25,7 +25,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
   <artifactId>hbase-rest</artifactId>
diff --git a/hbase-rsgroup/pom.xml b/hbase-rsgroup/pom.xml
index abc391e..7903981 100644
--- a/hbase-rsgroup/pom.xml
+++ b/hbase-rsgroup/pom.xml
@@ -24,7 +24,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-server/pom.xml b/hbase-server/pom.xml
index 5c746ab..bdeccf3 100644
--- a/hbase-server/pom.xml
+++ b/hbase-server/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
   <artifactId>hbase-server</artifactId>
diff --git a/hbase-shaded/hbase-shaded-check-invariants/pom.xml b/hbase-shaded/hbase-shaded-check-invariants/pom.xml
index b8fbdb4..ec26601 100644
--- a/hbase-shaded/hbase-shaded-check-invariants/pom.xml
+++ b/hbase-shaded/hbase-shaded-check-invariants/pom.xml
@@ -16,7 +16,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.1-SNAPSHOT</version>
     <relativePath>../..</relativePath>
   </parent>
   <artifactId>hbase-shaded-check-invariants</artifactId>
diff --git a/hbase-shaded/hbase-shaded-client/pom.xml b/hbase-shaded/hbase-shaded-client/pom.xml
index d080d4c..fb942cc 100644
--- a/hbase-shaded/hbase-shaded-client/pom.xml
+++ b/hbase-shaded/hbase-shaded-client/pom.xml
@@ -24,7 +24,7 @@
     <parent>
         <artifactId>hbase-shaded</artifactId>
         <groupId>org.apache.hbase</groupId>
-        <version>1.2.0-cdh5.16.0-SNAPSHOT</version>
+        <version>1.2.0-cdh5.16.1-SNAPSHOT</version>
         <relativePath>..</relativePath>
     </parent>
     <artifactId>hbase-shaded-client</artifactId>
diff --git a/hbase-shaded/hbase-shaded-server/pom.xml b/hbase-shaded/hbase-shaded-server/pom.xml
index 18c8e7b..11b9655 100644
--- a/hbase-shaded/hbase-shaded-server/pom.xml
+++ b/hbase-shaded/hbase-shaded-server/pom.xml
@@ -24,7 +24,7 @@
     <parent>
         <artifactId>hbase-shaded</artifactId>
         <groupId>org.apache.hbase</groupId>
-        <version>1.2.0-cdh5.16.0-SNAPSHOT</version>
+        <version>1.2.0-cdh5.16.1-SNAPSHOT</version>
         <relativePath>..</relativePath>
     </parent>
     <artifactId>hbase-shaded-server</artifactId>
diff --git a/hbase-shaded/pom.xml b/hbase-shaded/pom.xml
index 5472aea..47bb910 100644
--- a/hbase-shaded/pom.xml
+++ b/hbase-shaded/pom.xml
@@ -23,7 +23,7 @@
     <parent>
         <artifactId>hbase</artifactId>
         <groupId>org.apache.hbase</groupId>
-        <version>1.2.0-cdh5.16.0-SNAPSHOT</version>
+        <version>1.2.0-cdh5.16.1-SNAPSHOT</version>
         <relativePath>..</relativePath>
     </parent>
     <artifactId>hbase-shaded</artifactId>
diff --git a/hbase-shell/pom.xml b/hbase-shell/pom.xml
index e76644d..e2ae24c 100644
--- a/hbase-shell/pom.xml
+++ b/hbase-shell/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
   <artifactId>hbase-shell</artifactId>
diff --git a/hbase-spark/pom.xml b/hbase-spark/pom.xml
index e956c7c..26d7a8f 100644
--- a/hbase-spark/pom.xml
+++ b/hbase-spark/pom.xml
@@ -30,7 +30,7 @@
     <parent>
         <artifactId>hbase</artifactId>
         <groupId>org.apache.hbase</groupId>
-        <version>1.2.0-cdh5.16.0-SNAPSHOT</version>
+        <version>1.2.0-cdh5.16.1-SNAPSHOT</version>
         <relativePath>..</relativePath>
     </parent>
     <artifactId>hbase-spark</artifactId>
diff --git a/hbase-testing-util/pom.xml b/hbase-testing-util/pom.xml
index fcb2f32..8a97d60 100644
--- a/hbase-testing-util/pom.xml
+++ b/hbase-testing-util/pom.xml
@@ -23,7 +23,7 @@
     <parent>
         <artifactId>hbase</artifactId>
         <groupId>org.apache.hbase</groupId>
-        <version>1.2.0-cdh5.16.0-SNAPSHOT</version>
+        <version>1.2.0-cdh5.16.1-SNAPSHOT</version>
         <relativePath>..</relativePath>
     </parent>
     <artifactId>hbase-testing-util</artifactId>
diff --git a/hbase-thrift/pom.xml b/hbase-thrift/pom.xml
index 11e87a2..2c6a5dd 100644
--- a/hbase-thrift/pom.xml
+++ b/hbase-thrift/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.16.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.16.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
   <artifactId>hbase-thrift</artifactId>
diff --git a/pom.xml b/pom.xml
index b731609..59af38d 100644
--- a/pom.xml
+++ b/pom.xml
@@ -32,14 +32,14 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>cdh-root</artifactId>
-    <version>5.16.0-SNAPSHOT</version>
+    <version>5.16.1-SNAPSHOT</version>
     <relativePath/>
     <!-- no parent resolution -->
   </parent>
   <groupId>org.apache.hbase</groupId>
   <artifactId>hbase</artifactId>
   <packaging>pom</packaging>
-  <version>1.2.0-cdh5.16.0-SNAPSHOT</version>
+  <version>1.2.0-cdh5.16.1-SNAPSHOT</version>
   <name>Apache HBase</name>
   <description>
     Apache HBase is the Hadoop database. Use it when you need
-- 
1.7.9.5

